zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/ZebedeeCollectionReader.java

```diff
@@ -24,48 +22,54 @@
             "error constructing ZebedeeCollectionReader zebedee instance required but was null";
     static final String PERMISSIONS_SERVICE_NULL_ERR =
             "error constructing ZebedeeCollectionReader perissions service required but was null";
+    static final String USERS_SERVICE_NULL_ERR =
+            "error constructing ZebedeeCollectionReader users service required but was null";
     static final String KEYRING_NULL_ERR =
             "error constructing ZebedeeCollectionReader keyring requred but was null";
```

Comment:
Typo here (also in the ZebedeeCollectionWriter)
```suggestion
            "error constructing ZebedeeCollectionReader keyring required but was null";
```